### PR TITLE
[TASK] Add more development documentation

### PR DESCRIPTION
**Pull request by Starkmann**
Resolves:  #1393

### CODE REVIEW

File: Documentation/Environment.rst [hunk 1]
```diff
@@ -8,3 +8,63 @@ Development environment
 
 You can run the code quality checks and automated tests locally (using a
 local PHP, Composer, and database) or using runTests.sh.
+
+To kickstart the project we suggest the usage of the `TYPO3-testing-distribution <https://github.com/oliverklee/TYPO3-testing-distribution/>`__ by Oliver Klee
```
Comment (oliverklee) [inline Documentation/Environment.rst:12]: Please generally  let's keep the lines in the RST files to below 80 characters if possible.

File: Documentation/Environment.rst [hunk 2]
```diff
@@ -8,3 +8,63 @@ Development environment
...
+
+.. index:: Folder structure
+.. code-block:: bash
+
+    git\
+        TYPO3-testing-distribution
+        tea
+
+
+Inside the testing distribution there is a file **docker-compose.extensions.yaml.template**
```
Comment (oliverklee) [inline Documentation/Environment.rst:38]: This should be marked as code, not as bold.

File: Documentation/Environment.rst [hunk 3]
```diff
@@ -8,3 +8,63 @@ Development environment
...
+.. code-block:: bash
+
+    ddev start
+    ddev composer install
+    ddev install-typo3
+    ddev db-import
+
+After that you should be able to access the frontend:
+
+https://typo3-testing-12-4.ddev.site:8081/tea/
```
Comment (oliverklee) [inline Documentation/Environment.rst:70]: I propose to recommend `ddev launch` instead.

**Review by oliverklee — CHANGES_REQUESTED** [review]

**Comment by Starkmann** [comment]
Running ./Build/Scripts/runTests.sh -s functional native:

real	0m8,405s
user	0m0,097s
sys	0m0,128s

Running ./Build/Scripts/runTests.sh -s functional with paratest (without -f):

real	0m6,582s
user	0m0,106s
sys	0m0,097s

File: Documentation/Environment.rst [hunk 4]
```diff
@@ -9,9 +9,10 @@ Development environment
 You can run the code quality checks and automated tests locally (using a
 local PHP, Composer, and database) or using runTests.sh.
 
-To kickstart the project we suggest the usage of the `TYPO3-testing-distribution <https://github.com/oliverklee/TYPO3-testing-distribution/>`__ by Oliver Klee
-as development environment. The distribution comes with a frontend, example data and predefined
-plugins.
+To kickstart the project we suggest the usage of the
```
Comment (oliverklee) [inline Documentation/Environment.rst:12]: There should be a comma after "project".

File: Documentation/Environment.rst [hunk 5]
```diff
@@ -35,7 +37,12 @@ You can organize the folder structure as you wish, but lets say your folder stru
         tea
 
 
-Inside the testing distribution there is a file **docker-compose.extensions.yaml.template**
+Inside the testing distribution there is a file
+
+.. code-block:: bash
```
Comment (oliverklee) [inline Documentation/Environment.rst:42]: I was thinking more of inline code highlighting like this:

```rst
Inside the testing distribution, there is a file :file:`docker-compose.extensions.yaml.template`
which mounts the used extensions. This file need to be renamed and adjusted.
```

**Review by oliverklee — CHANGES_REQUESTED** [review]

File: Documentation/Environment.rst (hunk 4 above)
Comment (oliverklee) [reply Documentation/Environment.rst]: Done.

**Pull request merged**